spec/javascripts/app/views/help_view_spec.js

```diff
@@ -134,4 +141,28 @@ describe("app.views.Help", function(){
       });
     });
   });
+
+  describe("chat section", function(){
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        gon.chatEnabled = true;
```

Comment:
I don't think you should. There is no need to change `gon.chatEnabled` on a normal setup after loading the page so `this.chat_enabled` doesn't have to change either.

`this.chat_enabled` doesn't change after loading the page because you copy the value of `gon.chatEnabled` on initialization and don't change `this.chat_enabled` afterwards. (which is fine as I already wrote) You just have to make sure that in this particular case `this.chat_enabled` changes which you can do by adding `this.view = new app.views.Help();` here.